base/deprecated.jl

```diff
@@ -496,3 +496,19 @@ function to_index{T<:Real}(A::AbstractArray{T})
     depwarn("indexing with non Integer AbstractArrays is deprecated", :to_index)
     Int[to_index_nodep(x) for x in A]
 end
+
+function float_isvalid{T<:Union(Float32,Float64)}(s::AbstractString, out::Array{T,1})
+    tf = tryparse(T, s)
+    isnull(tf) || (out[1] = get(tf))
+    !isnull(tf)
+end
```

Comment:
Why no `depwarn` here?

A comment refering to `#10543` would also be great so that it is easier to see where a deprecation is comming from (without digging into git history).